app/src/main/report/components/ReportsList/ReportsListSorting.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { connect } from 'react-redux';
+import { Dispatch, Action } from "redux";
+
+import {reportsActions} from "../../actions/reportsActions";
+import {ReportsSortingFields} from "../../actionTypes/ReportsActionsTypes";
+
+interface ReportsListSortingProps {
+    sort: (value: ReportsSortingFields) => void;
+}
+
+export const ReportsListSortingComponent: React.StatelessComponent<ReportsListSortingProps> = (props: ReportsListSortingProps) => (
```

Comment:
Oh nice, didn't know about this `StatelessComponent` type, that's something we can use in a lot of places!